Clear coldkey swap (#217)
latent-to/developer-docs: docs/keys/coldkey-swap.md

Because they are such sensitive operations a security perspective, coldkey swaps unfold in several careful stages:

1. **Initiation/Announcement**

In the first step, the coldkey owner initiates the swap by making an announcement on the blockchain that the swap will occur. This triggers a mandatory waiting period, during which the wallet is locked to prevent operations such as transfers or staking. During this phase, the coldkey can only execute or dispute a swap.

At this initiation step, the coldkey owner provides the destination wallet address, which remains private, as only a hash is published to the blockchain.

2. **Pending Period**

Next, a pending or lock-out period must elapse, during which the swap can be disputed but not finalized.
Currently, the waiting/locked period is **36,000 blocks** (~ **5 days**).

3. **Disputation or Finalization**
   1. [Disputing a coldkey swap](#dispute-a-coldkey-swap) prevents the execution of the swap and completely blocks the coldkey from performing any operations. At this point, the triumvirate is required to resolve the dispute. The coldkey private key is required to dispute a swap.

>    2. If the Pending Period expires without the swap being disputed, the coldkey owner must finalize the swap by again providing a hash of the destination coldkey. The blockchain verifies that this key matches the recorded hash before proceeding.

Reviewer: ```suggestion
   2. If the Pending Period expires without the swap being disputed, the coldkey owner must finalize the swap by providing the destination coldkey. It will be checked against the on-chain coldkey hash provided during announcement before proceeding.
```
